frontend/packages/console-app/src/components/quick-starts/data/serverless-application-quickstart.ts

```diff
@@ -8,163 +8,143 @@ export const serverlessApplicationQuickStart = {
-4. Click the route URL icon in the top right corner of the Knative Service. The application will open in a new tab.
-5. Close the new browser tab and go back to the Topology view.
+1. From the **Display Options** list at the top of the **Topology** view, click **Pod Count**.
+2. Wait for the Pod count to scale down to zero Pods. Scaling down may take a few minutes.
+3. Click the **Route URL** icon in the upper-right corner of the Knative Service panel. The application opens in a new tab.
+4. Close the new browser tab and return to the **Topology** view.
 
-You should be able to see that your application scaled up to 1 Pod to accommodate the traffic request when you accessed the application. After a few minutes, you should see your application scale back down to 0 pods.`,
+In the **Topology** view, you can see that your application scaled up to one Pod to accommodate your request.  After a few minutes, your application scales back down to zero Pods.
+`,
         review: {
           instructions: `#### To verify the application scaled down:
-1. Click on the pod donut inside your service. The badge at the top of the side panel should be (REV).
-2. Click on the **Details** tab in the side panel.
-3. Is the pod donut autoscaled to 0?
+1. Click the revision inside your service. The badges under the Pod ring and at the top of the side panel should be (REV).
+2. Click the **Details** tab in the side panel.
 
-Is the status Succeeded?`,
-          taskHelp: 'Try walking through the steps again',
+Is the Pod ring autoscaled to zero?`,
+          taskHelp: `This task isn’t verified yet. Try the task again, or [read more](https://docs.openshift.com/container-platform/4.6/applications/application_life_cycle_management/odc-viewing-application-composition-using-topology-view.html#odc-scaling-application-pods-and-checking-builds-and-routes_viewing-application-composition-using-topology-view) about this topic.`,
         },
         recapitulation: {
-          success: "You've just scaled up your application to accomodate a traffic request!",
-          failed: 'Try the steps again.',
+          success: `You just scaled up your application to accomodate a traffic request!`,
+          failed: `Try the steps again.`,
         },
       },
       {
-        title: `Wire an event source to your Knative Service`,
-        description: `### To wire an event source to your Knative Service:
-1. Hover over your service with your mouse. You should see a blue arrow.
-2. Click the blue arrow, then drag and drop the **(+)** anywhere outside the service.
+        title: `Connecting an event source to your Knative Service`,
+        description: `### To connect an event source to your Knative Service:
+1. On the **Topology** View, hover over your service with your cursor. You should see a blue arrow.
+2. Click and drag the blue arrow, then drop it anywhere outside the service.
 3. In the dropdown menu that appears, click **Event Source**.
```

Comment:
drop-down list

Reply:
ISG prefers avoiding "drop-down" entirely unless it's important to identify the type of menu or list.

Reply:
So should this remain unchanged?

Reply:
You'd hyphenate "dropdown" or remove it entirely. 

It's not super important, but it could be:

`* 3. In the drop-down list that appears, click **Event Source**.`
`* 3. In the list that appears, click **Event Source**.`

My preference is the last one, but it's an individual preference beyond that.